spec/requests/user_request_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
```

Comment:
![image](https://github.com/The-Normal-One1/Blog-App/assets/93039359/8fd1a6bd-8af5-49eb-8b2a-b36e22c64a30)

- [ ] It would be nice if you fix your tests 👍 because your tests are failing which shows that your app is not working as expected 😨 Kindly make them pass. If they are passing on your side then kindly reset your database to remove all the entries in the database. After cleaning everything run your tests and see 😊